homework/Vasili_Zasinets/AbstractMath/src/main/java/com/tutrit/java/quickstart/abstractmath/OperationExecutor.java

```diff
@@ -0,0 +1,10 @@
+package com.tutrit.java.quickstart.abstractmath;
+
+public class OperationExecutor {
+
+    void execute(MathOperation[] operations, double a, double b) throws DivisionByZeroException {
```

Comment:
if you pass b==0, some operation might not been executed after exception thrown. Try to revise you logic to implement recovering after throwing DivisionByZeroException.

Reply:
fixed